docs/source/user_guide/automl.ipynb

```diff
@@ -645,17 +1598,1393 @@
+      "Mode Baseline Binary Classification Pipeline:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 12.868\n",
+      "\n",
+      "*****************************\n",
+      "* Evaluating Batch Number 1 *\n",
+      "*****************************\n",
+      "\n",
+      "Elastic Net Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n",
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n",
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Logistic Regression Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n",
+      "\tHigh coefficient of variation (cv >= 0.2) within cross validation scores.\n",
+      "\tLogistic Regression Classifier w/ Imputer + Standard Scaler may not perform as estimated on unseen data.\n",
+      "\n",
+      "Search finished after 00:09            \n",
+      "Best pipeline: Logistic Regression Classifier w/ Imputer + Standard Scaler\n",
+      "Best pipeline Log Loss Binary: 0.076807\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n"
+     ]
+    }
+   ],
+   "source": [
+    "from dask.distributed import Client, LocalCluster\n",
+    "\n",
+    "from evalml.automl.engine import DaskEngine\n",
+    "\n",
+    "dask_engine_t2 = DaskEngine(Client(LocalCluster(processes=True, n_workers = 2)))\n",
+    "automl_dask_t2 = AutoMLSearch(X_train=X, y_train=y,\n",
+    "                      problem_type=\"binary\",\n",
+    "                      allowed_model_families=[ModelFamily.LINEAR_MODEL],\n",
+    "                      engine = dask_engine_t2)\n",
+    "automl_dask_t2.search(show_iteration_plot = False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-07-27T18:56:35.392897Z",
+     "start_time": "2021-07-27T18:56:30.633575Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/Caskroom/miniconda/base/envs/evalml_dev/lib/python3.9/site-packages/distributed/node.py:160: UserWarning:\n",
+      "\n",
+      "Port 8787 is already in use.\n",
+      "Perhaps you already have a cluster running?\n",
+      "Hosting the HTTP server on port 50005 instead\n",
```

Comment:
This might just be a warning message you got locally, but would be nice to clear and make sure
![image](https://user-images.githubusercontent.com/22552445/127228886-9920fb3f-4add-4ffc-a166-5f6de3322ad8.png)

Reply:
My hope is that the warning is supressed when the changes to `conf.py` are reverted.

Reply:
Yea,apparently this comes from not putting the LocalCluster instantiation inside a `if __name__ == "__main__":` block.  I think this will go away with #2561